std_msgs/Header header
uint16 nav_distance
uint16 nav_yaw
uint8 EndOfNav
#navigation calculated route
